### DISCUSSION TITLE
How to test an array of Objects contains at least one item ?

### DISCUSSION THREAD

**Discussion by olympien13** [postmanlabs/postman-flows#201] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/20594171/148993130-084871f1-537e-4cd7-ae3b-900710d8127f.png)
I wanna test if the key registeredServices contains or not one item

**Comment by olympien13** [comment]
How can we test of a List is Empty or not ? Which block should we use ?

**Reply by saswatds** [reply]
Ideally this should be possible with expressions. But I think we are missing a length operator, I will add it as part of the next release.

**Reply by Bullkn0x** [reply]
Length operator!  This may have answered my latest feature request

**Comment by olympien13** [comment, marked as answer]
Hi yep Indeed that would be appreciable. 
Thanks for your answer

**Reply by saswatds** [reply]
With this release you should be able to use use the `len` (length) operator https://github.com/postmanlabs/postman-flows/releases/tag/v22.01.2

**Comment by olympien13** [comment]
Excellent news. Will test then with the POSTMAN 9.9.4
